kong/db/declarative/init.lua

```diff
@@ -104,7 +104,7 @@ function Config:parse_string(contents, filename, accept, old_hash)
     local chunk, pok
     chunk, err = loadstring(contents)
     if chunk then
-      setfenv(chunk, {})
+      setfenv(chunk, {_G.os = _G.os})
```

Comment:
I think we should be extra careful here, since `os` includes things like `os.remove`. I assume you're exposing `os` to get access to `os.getenv`? If so, maybe let's add this environment instead `{ os = { getenv = os.getenv } }`?

Reply:
Are we reassigning `os` so as to remove any attribute (such as `remove`) other than `getenv`?
Also, are we specifying the parameter to `os.getenv` at runtime?

Reply:
I assumed this is what you were trying to do by exposing `os` (the syntax of your table declaration in your `setfenv` call is not correct).

Reply:
Sorry for the trouble caused. Saw a few examples with `_G.a` = `_G.a` but that's only true for assignment, not declaration. I will take more care the next time.

Since, we are not specifying any `path_string` to `os.getenv`, I am assuming this will be provided at runtime.